### PR TITLE
Updates to the Proxima Nova instructions page.

### PR DESCRIPTION
**Pull request by camikazegreen**

### CODE REVIEW

**Review by geijutsuka — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by mmunro-ltrr — DISMISSED** [review]

**Review by jwmoore1 — APPROVED** [review]

File: site/docs/2.0/content/font.md [hunk 1]
```diff
@@ -13,57 +13,51 @@ toc: true
...
 
-For ease of integration into web projects around campus, the Arizona Digital
-team manages a Typekit webfont project that can be referenced by your site.
+### Reference link
 
-#### Notes to consider when using the centrally mananged webfont project.
-Our license with Typekit allows anyone with a netid to create their own webfont
-project in Creative Cloud Typekit and use it in all of their web projects. Of
-course there are advantages and disadvantages to each approach.
+Arizona Bootstrap uses a Creative Cloud Library with shared governance to
```
Comment (joeparsons) [inline site/docs/2.0/content/font.md:22]: Is this accurate?  I thought we abandoned the idea of using Adobe's Creative Cloud Library feature and are instead just using straight-up typekit with a centrally managed shared login now...

Comment (trackleft) [reply site/docs/2.0/content/font.md:22]: Yes, the Creative Cloud Library feature doesn't work.

**Review by jwmoore1 — APPROVED** [review]

File: site/docs/2.0/content/font.md (hunk 1 above)
Comment (camikazegreen) [reply site/docs/2.0/content/font.md]: Does this description work?

File: site/docs/2.0/content/font.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ employees through a license with Adobe Typekit.
 
 ### Reference link
 
-Arizona Bootstrap uses a Creative Cloud Library with shared governance to
+Arizona Bootstrap uses Adobe Typekit with a centrally managed netid to
```
Comment (trackleft) [inline site/docs/2.0/content/font.md:22]: Why not shared governance?

**Review by mmunro-ltrr — APPROVED** [review]

File: site/docs/2.0/content/font.md (hunk 2 above)
Comment (joeparsons) [reply site/docs/2.0/content/font.md:22]: Arizona Bootstrap uses a centrally managed Adobe TypeKit account to

Comment (camikazegreen) [reply site/docs/2.0/content/font.md:22]: Arizona Bootstrap uses an Adobe Typekit account with shared governance to
manage our font reference links within Arizona Bootstrap.
?

Comment (joeparsons) [reply site/docs/2.0/content/font.md:22]: Sounds good.

**Review by trackleft — DISMISSED** [review]

**Review by trackleft — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Pull request merged**